tables/automl/automl_tables_dataset.py

```diff
@@ -79,23 +79,38 @@ def list_datasets(project_id, compute_region, filter_=None):
         print("Dataset id: {}".format(dataset.name.split("/")[-1]))
         print("Dataset display name: {}".format(dataset.display_name))
         metadata = dataset.tables_dataset_metadata
-        print("Dataset primary table spec id: {}".format(
-            metadata.primary_table_spec_id))
-        print("Dataset target column spec id: {}".format(
-            metadata.target_column_spec_id))
-        print("Dataset target column spec id: {}".format(
-            metadata.target_column_spec_id))
-        print("Dataset weight column spec id: {}".format(
-            metadata.weight_column_spec_id))
-        print("Dataset ml use column spec id: {}".format(
-            metadata.ml_use_column_spec_id))
+        print(
+            "Dataset primary table spec id: {}".format(
```

Comment:
(Optionally fix this) I'll just note that I prefer single-quotes over double-quotes and that our authoring guide suggests single quotes.